StablecoinResearch/Stablecoins/Model/experiments_refined/test_refined_model.py

```
@@ -0,0 +1,37 @@
+"""
+Quick test of the refined model to verify functionality
+"""
+import sys, os
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '../../../..')))
+
+# Import and test the classes
+exec(open('refined_depeg_attack_simulation.py').read())
```

Ruff (0.12.2)

Use of `exec` detected

(S102)